pkg/controller/operators/catalog/operator.go

```diff
@@ -1578,7 +1578,16 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 
 	ensurer := newStepEnsurer(kubeclient, crclient, dynamicClient)
 	r := newManifestResolver(plan.GetNamespace(), o.lister.CoreV1().ConfigMapLister(), o.logger)
-	b := newBuilder(kubeclient, dynamicClient, o.csvProvidedAPIsIndexer, r, o.logger)
+
+	// CRDs should be installed via the default OLM (cluster-admin) client and not the scoped client specified by the AttenuatedServiceAccount
+	// the StepBuilder is currently only implemented for CRD types
+	// TODO give the StepBuilder both OLM and scoped clients when it supports new scoped types
+	builderKubeClient, _, builderDynamicClient, err := o.clientAttenuator.AttenuateClientWithServiceAccount(nil)
```

Comment:
this can be a follow up - 

is there a related bug where we won't grant the operator permission to manage its own types because the scoped serviceaccount doesn't have rbac for the new types?

Reply:
I can check but I thought this would be expected behavior instead of a bug? Or am I missing something